### ISSUE TITLE
1.16.5 cant get bush in conjunction with the botany pots mod

### ISSUE THREAD

**Issue by GlauberBlox** [Mrbysco/OreBerries-Replanted#9]
### General Info

- [X] I am running a modpack
- [X] I can reproduce this issue consistently in single-player
- [X] I can reproduce this issue consistently in multi-player
- [X] I have searched for this issue previously and it was either (1) not previously reported, or (2) previously fixed and I am having the same problem.
- [ ] I am crashing and can provide my crash report(s)
- [X] I am using the latest version of the mod

### Forge version

36.2.0

### Minecraft version

1.16.5

### Mod version

0.1.4.1

### Java version

Java 8

### Issue Description

basically, well, i can only plant the xp berry at lapis lazuli in botany pots at 1.16.5, but, i can't get a bush too to replant in all other pots that i made, basically thats it, it only gives the berry but the bush not, to replant.

**Comment by Mrbysco** [comment]
I no longer actively develop for 1.16.5

**Comment by GlauberBlox** [comment]
bruh
